README.md

```diff
@@ -51,6 +51,22 @@ server.get({
   next()
 })
 
+server.get({
+  path: '/anythingErr/:id',
+  validation: {
+    params: Joi.object().keys({
+      id: Joi.number().min(0).required()
+    }).required()
+  },
+  errorResponder (transformedErr, req, res, next) {
+    req.ifError = transformedErr
+    return next()
+  }
+}, (req, res, next) => {
+  res.send(200, {err: req.ifError})
+  next()
+})
+
```

Comment:
`validationOptions` would probably be a better name for route-level validation settings since we already have `validation`.  What do you think about this approach?
```javascript
server.post({
  path: '/anything',
  validationOptions: {
    joiOptions: {
      allowUnknown: true
      // additional Joi options here
    },
    errorResponder: (transformedErr, req, res, next) => {
      req.ifError = transformedErr
      return next()
    },
    errorTransformer: (validationInput, joiError) => new restifyErrors.BadRequestError(),
  },
  validation: {
    body: Joi.object().keys({
      name: Joi.string().required()
    }).required()
  }
}, (req, res, next) => {
  res.send(201, {params: req.params})
  next()
})
```

Reply:
I notice that this way it gets more organized 👍